lib/taurus/core/tango/tangoattribute.py

```diff
@@ -1078,6 +1078,8 @@ def _decodeAttrInfoEx(self, pytango_attrinfoex=None):
         match = re.search("[^\.]*\.(?P<precision>[0-9]+)[eEfFgG%]", fmt)
         if match:
             self.precision = int(match.group(1))
+        elif re.match("%[0-9]*d", '%d'):
```

Comment:
I think that this line is wrong: it is static and always evaluates to `True `

Perhaps you intended to write the following?

```python
elif re.match("%[0-9]*d", fmt):`
```

Reply:
You are right